api/src/main/java/io/minio/MinioClient.java

```diff
@@ -3413,7 +3413,7 @@ private String putObject(String bucketName, String objectName, int length,
    */
   private void putObject(String bucketName, String objectName, Long size, Object data,
       Map<String, String> headerMap, ServerSideEncryption sse)
-    throws InvalidBucketNameException, NoSuchAlgorithmException, InsufficientDataException, IOException,
+    throws InvalidBucketNameException, NoSuchAlgorithmException, IOException,
```

Comment:
Isn't `UnsufficentDataException` going to be thrown any more for all the above methods?

Reply:
`InsufficientDataException` was redundant in the signature, It was coming twice. So removed the extra one.